README.md

```diff
@@ -82,16 +82,49 @@ Below are instructions for how to deploy this NOMAD Oasis distribution
     docker compose pull
     ```
 
-5. And run it with docker compose in detached (--detach or -d) mode
+5. (Optional) Deploy Oasis with HTTPS
 
+    Generate a self-signed SSL certificate (or use a trusted certificate authority if preferred):
     ```sh
-    docker compose up -d
+    mkdir ssl
+    openssl req -x509 -nodes -days 365 \
+      -newkey rsa:2048 \
+      -keyout ./ssl/selfsigned.key \
+      -out ./ssl/selfsigned.crt \
+      -subj "/CN=localhost"
     ```
 
-6. Optionally you can now test that NOMAD is running with
+    Update the `proxy` config in `docker-compose.yml` to use the HTTPS Nginx config instead of the HTTP one:
+    ```diff
+    - # HTTP
+    - - ./configs/nginx_http.conf:/etc/nginx/conf.d/default.conf:ro
 
+    + # HTTPS (you need to generate SSL certificate)
+    + - ./configs/nginx_https.conf:/etc/nginx/conf.d/default.conf:ro
+    + - ./ssl:/etc/nginx/ssl:ro  # generate your SSL certificate
+    ```
+
+    Also make sure port 443 is exposed:
+    ```yaml
+    ports:
+      - 80:80
```

Comment:
I think it should be either 80 or 443. It doesn't sound to safe to leave 80 open is you enable 433

Reply:
Ok, I see the 80->443 redirect in the config so technically this is fine. Only issue when the local IT doesn't allow you to use port 80 only 443, which is quite common, but we do not deal with it.